### ISSUE TITLE
User Defined MetaData wiped on load

### ISSUE THREAD

**Issue by JonathonMSmith** [pysat/pysat#374]
In version 2.1.0 an instrument with some meta data assigned will lose the user defined meta data after loading data.

1. Install pysat
2. Create Instrument
3. Download data if needed
4. Assign meta data to instrument and check that it was assigned properly
5. Load instrument data
6. Check that meta data is still there
7. See error: AttributeError: No attribute user_meta

**Expected behavior**
I expect the assigned meta data to be unchanged by pysat.

Code test used to produce these results:
```python
from datetime import datetime
import pysat

vefi = pysat.Instrument(platform='cnofs', name='vefi', tag='dc_b')
start = datetime(2010,1,1)
stop = datetime(2010,1,1)
vefi.bounds = (start, stop)
vefi.download(start, stop)
vefi.user_meta = 'somethingMeta'
print('user metadata is : '+vefi.user_meta)

vefi.load(date=start)

print('user metadata is : '+vefi.user_meta)
```
output from pysat version 2.1.0 (paths truncated for space)
```
Downloading data to:  /cnofs/vefi/dc_b/
Attempting to download file for 01 January 2010
Finished.
Updating pysat file list
pysat is searching for cnofs vefi dc_b files.
Found 1 of them.
Updating instrument object bounds.

user metadata is : somethingMeta

Returning cnofs vefi dc_b data for 01 January 2010
Traceback (most recent call last):
  File "/pysat-2.1.0-py3.7.egg/pysat/_instrument.py", line 401, in __getattr__
    return getattr(self.meta, name)
AttributeError: 'Meta' object has no attribute 'user_meta'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "meta_test.py", line 14, in <module>
    print('\nuser metadata is : '+vefi.user_meta+'\n')
  File "/pysat-2.1.0-py3.7.egg/pysat/_instrument.py", line 403, in __getattr__
    raise AttributeError("No attribute {}".format(name))
AttributeError: No attribute user_meta

```
output from pysat version 2.0.0, this is what I expect to happen in version 2.1.0 as well
```
Downloading data to:  /cnofs/vefi/dc_b/
Attempting to download file for 01 January 2010
Finished.
Updating pysat file list
pysat is searching for cnofs vefi dc_b files.
Found 1 of them.
Updating instrument object bounds.

user metadata is : somethingMeta

Returning cnofs vefi dc_b data for 01 January 2010

user metadata is : somethingMeta

```

**Desktop (please complete the following information):**
 - OS: CentOS 7
 - Python Version 3.7
 - Both version of pysat running in dedicated conda environments

**Additional context**
I  have only tried this for cnofs instruments 'ivm' and 'vefi'
This does not occur in pysat version 2.0.0.
I haven't been able to find the breaking change at a glance.
I will be going through the commit history to try and find what change exactly produced this behavior.

**Comment by rstoneback** [comment]
Thanks for the report! Corrected in pull #397.

**Comment by jklenzing** [comment]
Closing with merge of #397.
